pkgs/build-support/dotnet/add-nuget-deps/fetch-deps.sh

```diff
@@ -7,7 +7,7 @@ LOCKFILE_OUTPUT="$1"
 
 genericBuild
 
-nuget-to-json "${NUGET_PACKAGES%/}" | jq . >deps.json
+nuget-to-json "${NUGET_PACKAGES%/}" >deps.json
```

Comment:
``buildDotnetModule.fetch-deps: use json lockfiles by default`` seems to just have this change; not sure it actually relates to what is used by default.

It feels more like a "fixup" commit for the previous commit?

Reply:
This got mixed with the previous commit, all of `fetch-deps.sh` changes should be in this commit.

Reply:
Ah, stepping through I've noticed a few other commits have unrelated changes that should probably have been in different commits;

e.g. [`c6af33c` (#362278)](https://github.com/NixOS/nixpkgs/pull/362278/commits/c6af33c0fc018c52900a896b2d6b327b7c5eb29c) also contains changes to `mkNugetDeps`